WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
+    content = capture(&block) if block_given?
+    content.prepend("<strong>Note:</strong> ") if note
```

Comment:
Of course, prepend doesn't do the right thing here...

```
irb(main):008:0> "YIKES<".prepend("<strong>note:</strong>".html_safe)
=> "<strong>note:</strong>YIKES<"
```

Urg. Can we change this to an explicit string concatenation, rather than using `.prepend`